test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java

```diff
@@ -514,6 +514,7 @@ private Settings getSettings(int nodeOrdinal, long nodeSeed, Settings others) {
     public Collection<Class<? extends Plugin>> getPlugins() {
         Set<Class<? extends Plugin>> plugins = new HashSet<>(nodeConfigurationSource.nodePlugins());
         plugins.addAll(mockPlugins);
+        plugins.add(nodeConfigurationSource.telemetryPlugin());
```

Comment:
This is not need - should come out of `nodeConfigurationSource.nodePlugins()`

Reply:
Replied in other comment https://github.com/opensearch-project/OpenSearch/pull/8444/files/15dfdf0c48b8d188258acba5e88d4e13fa7f594d#r1264131603

Reply:
ack.